lang/cs/Org.Apache.REEF.IMRU/OnREEF/Client/REEFIMRUClient.cs

```diff
@@ -50,6 +51,7 @@ internal sealed class REEFIMRUClient : IIMRUClient
         private readonly IREEFClient _reefClient;
         private readonly JobSubmissionBuilderFactory _jobSubmissionBuilderFactory;
         private readonly AvroConfigurationSerializer _configurationSerializer;
+        private IDriverHttpEndpoint _httpEndPoint;
```

Comment:
It cannot be readonly since it is assigned in Submit function

On Mon, Oct 19, 2015 at 10:56 AM, Julia Wang <notifications@github.com>
wrote:

> In lang/cs/Org.Apache.REEF.IMRU/OnREEF/Client/REEFIMRUClient.cs
> <https://github.com/apache/incubator-reef/pull/573#discussion_r42403742>:
>
> > @@ -50,6 +51,7 @@ internal sealed class REEFIMRUClient : IIMRUClient
> >          private readonly IREEFClient _reefClient;
> >          private readonly JobSubmissionBuilderFactory _jobSubmissionBuilderFactory;
> >          private readonly AvroConfigurationSerializer _configurationSerializer;
> > +        private IDriverHttpEndpoint _httpEndPoint;
>
> readonly
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-reef/pull/573/files#r42403742>.
>